Sources/Apodini/Semantic Model Builder/InterfaceExporter/Validator.swift

```diff
@@ -109,8 +109,17 @@ private class EndpointValidator<I: InterfaceExporter, H: Handler>: Validator {
                 throw error
             }
         }
-        
-        return ValidatedRequest(for: exporter, with: request, using: output, on: endpoint, running: eventLoop)
+
+        if let requestWithRemote = request as? WithRemote {
+            return ValidatedRequest(for: exporter,
+                                    with: request,
+                                    using: output,
+                                    on: endpoint,
+                                    running: eventLoop,
+                                    remoteAddress: requestWithRemote.remoteAddress)
+        } else {
+            return ValidatedRequest(for: exporter, with: request, using: output, on: endpoint, running: eventLoop, remoteAddress: nil)
+        }
```

Comment:
```suggestion
        let remoteAddress = (request as? WithRemote)?.remoteAddress
        return ValidatedRequest(for: exporter, with: request, using: output, on: endpoint, running: eventLoop, remoteAddress: remoteAddress)
```

Just a little bit shorter 🙂

Reply:
I think swift lint won't allow this 😄